src/cpp/CMakeLists.txt

```diff
@@ -191,49 +191,21 @@ set(${PROJECT_NAME}_source_files
     rtps/reader/StatefulPersistentReader.cpp
     rtps/persistence/PersistenceFactory.cpp
 
-    # old discovery-server sources
+    rtps/builtin/discovery/database/DiscoveryDataBase.cpp
+    rtps/builtin/discovery/database/DiscoveryParticipantsAckStatus.cpp
+    rtps/builtin/discovery/database/DiscoveryParticipantInfo.cpp
+    rtps/builtin/discovery/database/DiscoverySharedInfo.cpp
+    rtps/builtin/discovery/database/backup/SharedBackupFunctions.cpp
 
     rtps/builtin/discovery/participant/timedevent/DSClientEvent.cpp
-    rtps/builtin/discovery/participant/timedevent/DServerEvent.cpp
     rtps/builtin/discovery/participant/PDPClient.cpp
+    rtps/builtin/discovery/endpoint/EDPClient.cpp
+    rtps/builtin/discovery/participant/timedevent/DServerEvent.cpp
```

Comment:
Even though is not done anywhere, I think it'd be good to sort these alphabetically. Just limit is to the DS files

Reply:
Still applies